eslint/README.md

```diff
@@ -0,0 +1,3 @@
+The files in this directory were copied from https://github.com/Khan/khan-linter/tree/60889f7a1c86e9eb7f3154184a89504a3d2163f8.
```

Comment:
And they're all un-modified, right? We may want to explicitly state that they shouldn't be modified and if we want to add a rule we should modify `.eslintrc`, instead.